### PR TITLE
Updating sim recon cmds

### PR DESCRIPTION
**Pull request by cfauci**
combined the simEvol, reconSeq and reconAccuracy cmds so they were easier to use. Includes updates to simulate.go to fix bugs found while doing testing on larger trees.

### CODE REVIEW

File: cmd/simRecon/simRecon.go [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+	"fmt"
+	"github.com/vertgenlab/gonomics/expandedTree"
+	"github.com/vertgenlab/gonomics/fasta"
+	"github.com/vertgenlab/gonomics/fileio"
+	"github.com/vertgenlab/gonomics/reconstruct"
+	"github.com/vertgenlab/gonomics/simulate"
+	"log"
+)
+
+func SimulateEvol(rootFastaFile string, treeFile string, gp string, simOutFile string, leafOutFile string) {
```
Comment (edotau) [inline cmd/simRecon/simRecon.go:14]: I was just thinking if this was named SimulateEvolve would only be an extra 2 key strokes so the full name wouldn't be too much longer (just food for thought)

**Review by edotau — COMMENTED** [review]
This looks great. Two small things related to the git branch:
1. when I checkout your branch and run `go test ./... ` I am getting a few errors (unrelated to this check in, but within your git branch), but for some reason I'm getting errors that I think these were fixed recently so could I ask you to:
```
git checkout main
git pull
git checkout updatingSimReconCmds
git merge main
```
these commands should update your branch and fix the errors without you needing to do anything

2. I am getting a compile error in the reconstruct package, it says:
```
# github.com/vertgenlab/gonomics/reconstruct [github.com/vertgenlab/gonomics/reconstruct.test]
./reconstruct_test.go:26:20: not enough arguments in call to simulate.Simulate
	have (string, *expandedTree.ETree)
	want (string, *expandedTree.ETree, string)
```

**Review by craiglowe — DISMISSED** [review]
Great Work!

**Comment by edotau** [comment]
LGTM!

**Review by edotau — DISMISSED** [review]
LGTM!!

File: reconstruct/reconstruct_test.go [hunk 2]
```diff
@@ -23,7 +23,7 @@ func Test_reconstruct(t *testing.T) {
 			log.Fatal("Couldn't read file")
 		}
 		fasta.Write("RandGeneOutput.fasta", simulate.RandGene("test", test.length, GCcontent)) //galGal6 GC
-		simulate.Simulate("RandGeneOutput.fasta", tre)
+		simulate.Simulate("RandGeneOutput.fasta", tre, "newTestFiles/genePred.gp")
```
Comment (craiglowe) [inline reconstruct/reconstruct_test.go:26]: There is a convention to put test files in a folder called "testdata", which the go tools know to ignore when looking for code.  Could you change this from newTestFiles to testdata?

File: simulate/simulate_test.go [hunk 3]
```diff
@@ -35,7 +33,7 @@ var MutateSeqTests = []struct {
 	branchLength float64
 	gp           string
 }{
-	{"testData/debug.fasta", 0.5, "testData/debug.gp"}, //branch length of 1 gives higher chance of returning a new base so you can see a difference even with a short sequence
+	{"testData/longDebug.fasta", 0.5, "testData/longDebug.gp"}, //branch length of 1 gives higher chance of returning a new base so you can see a difference even with a short sequence
```
Comment (craiglowe) [inline simulate/simulate_test.go:36]: Can we also change this to testdata as well? (all lowercase)

**Review by craiglowe — COMMENTED** [review]
Looks great!  One small request to use the "testdata" convention for directories holding testing files.

**Review by craiglowe — APPROVED** [review]
Wonderful!

**Review by edotau — APPROVED** [review]
LGTM!

**Review by DanielSnellings — APPROVED** [review]

**Pull request merged**